compiler/codegen/OMRCodeGenerator.hpp

```diff
@@ -1140,6 +1140,9 @@ class OMR_EXTENSIBLE CodeGenerator
    void apply32BitLabelTableRelocation(int32_t * cursor, TR::LabelSymbol *);
    bool supportsMergingGuards();
 
+   // This query is used to check if a method call is implemented using an internal runtime
+   // helper routine (ex. a j2iTransition)
+   bool callUsesHelperImplementation(TR::Symbol *sym) { return false; }
```

Comment:
Could we write Doxygen documentation for this? Also this API seems to be only used on Z. Can we migrate the definition into the Z code generator?

Reply:
See ae81f71c73a000436ccecfb8b76f8c8e04f19038 for these changes.